frontend/src/components/labels/index.tsx

```diff
@@ -158,7 +158,7 @@ export const Labels = () => {
           if (deleteDialogCtl?.data) deleteLabel(deleteDialogCtl.data);
         }}
       />
-      <PageHeader icon={FolderIcon} title={t("title.labels")}>
+      <PageHeader icon={LabelIcon} title={t("title.labels")}>
```

Comment:
Can you please use the same icon used in the sidebar ?
<img width="295" alt="image" src="https://github.com/user-attachments/assets/37e2336b-036e-46f1-8562-6dc949c932d2">

Reply:
yhe sure

Reply:
@marrouchi updated the pr

![Screenshot from 2024-09-22 12-42-57](https://github.com/user-attachments/assets/3663e391-e413-49b4-9294-cbec7e23a126)